modules/015-admission-policy-engine/docs/GATEKEEPER-CR.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "The admission-policy-engine module: Custom Resources (by Gatekeeper)"
+---
+
+## Mutation Custom Resouces
+
+[Reference](https://open-policy-agent.github.io/gatekeeper/website/docs/mutation/#mutation-crds)
+
+Provide a configurable set of policies for modifying Kubernetes resources at the time they are deployed.
```

Comment:
Do we want to add some description about `reinvocationPolicy: IfNeeded` used by default in the Deckhouse with the link to the official documentation.

@z9r5